### ISSUE TITLE
The two calendars display vertically, even if there's plenty of space

### ISSUE THREAD

**Issue by tarockx** [dangrossman/daterangepicker#1977]
In all the demos on the DateRangePicker homepage I see that the two calendars (when selecting a range) are side by side.

However, when I try to use it on my page, they get "stacked" vertically, one on top of the other.
I could understand this on small screens, but as you can see from the screenshot I attached, there is plenty of space on the page to show them side by side.

Am I missing some setting? How do I force it to display the calendars side by side?

![image](https://user-images.githubusercontent.com/10919460/60575062-293e5580-9d7b-11e9-98ab-ed623715964c.png)

**Comment by dangrossman** [comment]
It's not a setting. Looks like they're probably in a container that's not wide enough to display side by side. Can't tell more from a screenshot.

**Comment by tarockx** [comment]
Thanks for your reply.
Do you mean the `parentEl` option? I tried setting it to `body` but nothing changes. The html is correctly added to the body element, but it's still vertical:

![image](https://user-images.githubusercontent.com/10919460/60647339-cbbf0d00-9e3d-11e9-8ae3-dab6798aec38.png)

**Comment by tarockx** [comment]
Ok, I think I found the issue. The two `.drp-calendar` divs have a `left` and `right` class, those classes are in use by the UI framework I'm using on the page and they're messing with daterangepicker.

As a workaround, I edited `daterangepicker.css`, modifying the following rule:
```
.daterangepicker .ranges, .daterangepicker .drp-calendar {
    float: left !important;
  }
```
(I added `!important` to the existing rule)
Now it works perfectly!
